crates/bevy_ecs/src/query/filter.rs

```diff
@@ -631,10 +650,10 @@ impl_flag_filter!(
     Mutated,
     MutatedState,
     MutatedFetch,
-    ComponentFlags::MUTATED
+    ComponentCounters::is_mutated
 );
 
-impl_flag_filter!(
+impl_counter_filter!(
     /// Filter that retrieves components of type `T` that have been added or mutated since the start of the frame
```

Comment:
See comments on docstring for `Added` above.

Reply:
In particular, `Changed` is very useful for synchronizing components, especially now that it's reliable.